x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/RealmSettingsTests.java

```diff
@@ -261,7 +260,7 @@ public void testSettingsWithMultipleRealmsValidatesSuccessfully() throws Excepti
         if (withCacheSettings) {
             builder.put("cache.ttl", randomPositiveTimeValue())
                     .put("cache.max_users", randomIntBetween(1_000, 1_000_000))
-                    .put("cache.hash_algo", randomFrom(HASH_ALGOS));
+                .put("cache.hash_algo", randomFrom(CACHE_HASHING_ALGOS));
```

Comment:
indentation is off here. Either way works for me with regards to changing one line vs all of them